src/GraphQL.NewtonsoftJson/GraphQLRequestListJsonConverter.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using GraphQL.Transport;
+using Newtonsoft.Json;
+
+namespace GraphQL.NewtonsoftJson
+{
+    /// <summary>
+    /// A custom JsonConverter for reading or writing a <see cref="GraphQLRequest"/> object.
```

Comment:
```suggestion
    /// A custom JsonConverter for reading a <see cref="GraphQLRequest"/> object. Doesn't support writing.
```

Reply:
Updated similar to STJ comment, which was more accurate.